libraries/chain/db_notify.cpp

```diff
@@ -351,6 +376,8 @@ static void get_relevant_accounts( const object* obj, flat_set<account_id_type>&
               accounts.insert( aobj->bidder );
               break;
            }
+           case impl_escrow_object_type:
```

Comment:
Need code here.

Reply:
added code at https://github.com/bitshares/bitshares-core/pull/866/commits/1581691802232c9001a9a855751c3f2174dfa2f4
wondering if we should take the opportunity to change all the assert here to FC_ASSERT ...